stable/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 1.0.3
+version: 1.0.4
```

Comment:
Since it is a new feature, it may be nice to bump the minor version instead of patch.